### ISSUE TITLE
Add EVM Opcodes to EthOn

### ISSUE THREAD

**Issue by jo-tud** [ConsenSys/EthOn#10]
See yellow paper, _Appendix H: Virtual Machine Specification_
There are 130 opcodes. If they are modeled, together with some replenishing properties, EVM stack traces could be modeled.

Example ontology for opcodes (Manchester Syntax):
```
Class: EVMInstruction

    Annotations: 
        rdfs:comment "An instruction executable by the Ethereum Virtual Machine."@en,
        rdfs:label "EVM instruction"@en

Class: 30sInstruction

    Annotations: 
        rdfs:label "Environmental Information Instruction"@en
    
    SubClassOf: 
        EVMInstruction
    
Class: ORIGIN

    SubClassOf: 
        30sInstruction,
        addToStack some {"1"^^xsd:int},
        instructionCode some {"32"^^xsd:hexBinary},
        instructionMnemonic some {"ORIGIN"},
        removeFromStack some {"0"^^xsd:int}

DataProperty: addToStack

    Annotations: 
        rdfs:comment "rdfs:comment \"The number of items added to the stack with an operation.\""
    
    Domain: 
        EVMInstruction
    
    Range: 
        xsd:nonNegativeInteger
    
    SubPropertyOf: 
        owl:topDataProperty
    
DataProperty: instructionCode

    Annotations: 
        rdfs:comment "The hex value that represents an instruction from the EVM instruction set."@en
    
    Domain: 
        EVMInstruction
    
    Range: 
        xsd:hexBinary
    
    SubPropertyOf: 
        owl:topDataProperty
    
DataProperty: instructionMnemonic

    Annotations: 
        rdfs:comment "The string mnemonic of an EVM instruction."@en
    
    Domain: 
        EVMInstruction
    
    Range: 
        xsd:string
    
    SubPropertyOf: 
        owl:topDataProperty
    
DataProperty: removeFromStack

    Annotations: 
        rdfs:comment "The number of items removed from the stack with an operation."
    
    Domain: 
        EVMInstruction
    
    Range: 
        xsd:nonNegativeInteger
    
    SubPropertyOf: 
        owl:topDataProperty
    
```

**Comment by kumavis** [comment]
@jo-tud what is a `30sInstruction`

**Comment by jo-tud** [comment]
"30s Instruction" is just a way the yellow paper organizes instruction groups by hex range. See p. 24 of yellow paper.

**Comment by jo-tud** [comment]
fixed with Contracts extension ca8cc3a2249e0df53e40b5026d3064f31fa89bdb
